recipes/magpylib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "magpylib" %}
+{% set version = "1.0.2b0" %}
+{% set pypiSourceChecksum = 'fde2f8641a265461a9eb50259c5fccdf82822b57a441e3eae7a99f2861f3022e' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ pypiSourceChecksum }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python>=3.6
+    - pip
+  run:
+    - python
+    - matplotlib>=3.1
+    - numpy>=1.16
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: http://github.com/magpylib/magpylib
+  license: AGPL-3.0
+  license_family: AGPL
+  license_file: LICENSE
+  summary: 'A simple, user friendly Python 3 toolbox for calculating magnetic fields from permanent magnets and current distributions.'
+
+  description: |
+    magpylib provides simple and easy to use classes for calculating magnetic
+    fields depending only on numpy and matplotlib. The core of the library is
+    the source class which can represent permanent magnets, current
+    distributions or magnetic moments in a 3D space. The library provides simple
+    ways to generate such source objects, to manipulate them geometrically, to
+    group several sources into a Collection system and to calculate the
+    resulting fields of such systems. The systems may also be quickly visualized
+    with a matplotlib based 3D plot method.
+  doc_url: https://magpylib.readthedocs.io/en/latest/
+  dev_url: https://github.com/magpylib/magpylib
+
+extra:
+  recipe-maintainers:
+    - OrtnerMichael
+    - lucasgcb
```

Comment:
@lucasgcb Are you happy to be listed as a maintainer of this recipe?

Reply:
Yes, I am.

Reply:
Yes, I am.